src/resources/Limits/Limits.ts

```diff
@@ -27,6 +30,16 @@ export default class Limits extends Resource {
         return this.api.get<AllLimitsModel>(this.buildPath(Limits.getBaseUrl(), {limitType: limitType}));
     }
 
+    /**
+     * Shows the status of a specific limit in an organization
+     *
+     * @param sectionName The name of the target license section
+     * @param limitKey The unique identifier of the target limit status to show
+     */
+    getSpecificLimitStatus(sectionName: LicenseSection, limitKey: string) {
+        return this.api.get<LimitModel>(`${Limits.getSpecificLimit(sectionName, limitKey)}`);
```

Comment:
Could you just do
```suggestion
        return this.api.get<LimitModel>(Limits.getSpecificLimit(sectionName, limitKey));
```
since _getSpecificLimit_ already returns a string literal?

Reply:
Yep, you're right 👍 !